datasets/za/pmg_legislators/za_pmg_legislators.yaml

```
@@ -0,0 +1,56 @@
+title: South Africa National and Provincial Legislature and Executives
+entry_point: crawler.py
+prefix: za-pmg-leg
+coverage:
+  frequency: daily
+  start: 2024-08-08
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: > 
+ Current members of the National Assembly, National Council of Provinces,
+ and provincial legislatures and executives in South Africa.
+description: |
+    This dataset includes information about current members of South Africa's
+    National Assembly, National Council of Provinces, provincial legislatures,
+    and provincial executives. It covers various positions in the National
+    Legislature and Executive branch.
+publisher: 
+  name: Parliamentary Monitoring Group (PMG)
+  description: |
+   The Parliamentary Monitoring Group (PMG) provides comprehensive details
+   about parliamentary activities and various stakeholders in South Africa.
+  url: https://pmg.org.za/
+  official: true
```

```suggestion
  official: false
```

PMG is a civil society organisation and not an official source (even though they provide better access to parliamentary proceedings than government itself)